code/addons/docs/src/blocks/controls/Object.tsx

```diff
@@ -252,16 +252,14 @@ export const ObjectControl: FC<ObjectProps> = ({ name, value, onChange, argType
     <Wrapper aria-readonly={readonly}>
       {isObjectOrArray && (
         <RawButton
-          role="switch"
-          aria-checked={showRaw}
-          aria-label={`Edit the ${name} properties in text format`}
+          pressed={showRaw}
+          ariaLabel={`Edit the ${name} properties in text format`}
           onClick={(e: SyntheticEvent) => {
             e.preventDefault();
             setShowRaw((isRaw) => !isRaw);
           }}
         >
-          {showRaw ? <EyeCloseIcon /> : <EyeIcon />}
-          <span>RAW</span>
+          <span>Raw JSON</span>
```

Comment:
```suggestion
          Raw JSON
```

no need for span if we remove the icon